interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp_solver.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, qp_dims, arg):
 				'var': __hpipm.d_ocp_qp_ipm_get_ric_p
 			},
 			'ric_K': {
-				'n_row': __hpipm.d_ocp_qp_dim_get_nu,
-				'n_col': __hpipm.d_ocp_qp_dim_get_nx,
+				'n_row': __hpipm.d_ocp_qp_dim_get_nx,
+				'n_col': __hpipm.d_ocp_qp_dim_get_nu,
```

Comment:
It was correct at it was before, K is the state feedback matrix of size nu times nx.
It is probably a bit confusing, but `n_row` is the number of rows, not the length of each row.
Consistently with that, the (column) vectors 'lr', 'p' and 'k' have `n_row` but not 'n_col'.

Reply:
If you found numerically from the example that it was wrong as it was before, it may be that the issue is this: HPIPM writes back the result in column-major, but numpy matrices are row-major by default.
If this is the case, either the feedback matrix is saved in a temporary matrix and transposed into the actual one, or the zeros numpy matrix is set to column-major before getting the data from C, e.g. like in https://github.com/giaf/hpipm/blob/master/interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py#L75

Reply:
Column- vs. row-major was indeed the problem! thanks :+1: